docker/initial-data.json

```diff
@@ -68,6 +69,7 @@
     },
     "resource": {},
     "organization_tag": {},
+    "theme": {},
```

Comment:
Please create a default theme here and in the example data so `default_theme_id` can be required - it is not valid to have no theme

Reply:
Add a theme to the inital data and the example data.